src/ui/public/notify/__tests__/notifier.js

```diff
@@ -229,16 +241,17 @@ describe('Notifier', function () {
       // destroy the default custom notification, avoid duplicate handling
       customNotification.clear();
 
-      const explicitLifetimeParams = _.defaults({ lifetime: 20000 }, customParams);
-      customNotification = notifier.custom(customText, explicitLifetimeParams);
+      const overrideParams = _.defaults({ lifetime: 20000, truncationLength: 1000 }, customParams);
+      customNotification = notifier.custom(customText, overrideParams);
 
       expect(customNotification).to.have.property('type', 'info'); // default
-      expect(customNotification).to.have.property('title', explicitLifetimeParams.title); // passed in
-      expect(customNotification).to.have.property('lifetime', explicitLifetimeParams.lifetime); // passed in
+      expect(customNotification).to.have.property('title', overrideParams.title); // passed in thru customParams
+      expect(customNotification).to.have.property('truncationLength', 1000); // passed in thru overrideParams
+      expect(customNotification).to.have.property('lifetime', overrideParams.lifetime); // passed in thru overrideParams
 
-      expect(explicitLifetimeParams.type).to.be(undefined);
-      expect(explicitLifetimeParams.title).to.be.a('string');
-      expect(explicitLifetimeParams.lifetime).to.be.a('number');
+      expect(overrideParams.type).to.be(undefined);
+      expect(overrideParams.title).to.be.a('string');
+      expect(overrideParams.lifetime).to.be.a('number');
     });
```

Comment:
Does it make sense to add an assertion for `overrideParams.truncationLength` to be a number?